lib/test/t_IntegrationExpansion_std.cxx

```diff
@@ -0,0 +1,154 @@
+//                                               -*- C++ -*-
```

Comment:
The reference C++ unit test file `lib/test/t_IntegrationExpansion_std.expout` is missing in the PR.